Cheatsheet.md

```diff
@@ -0,0 +1,504 @@
+# Cheatsheet
+## Commonly used functions on [Array](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Arrays)
+### [1. map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map)
+
+**Description:**

```

Comment:
These should all be 4th level headings, not just bold (remove the colon too)

Reply:
Bold is not working with fourth level headings. So, they are now unbolded fourth level headings.